builder/openstack/step_create_volume.go

```diff
@@ -82,6 +82,11 @@ func (s *StepCreateVolume) Run(ctx context.Context, state multistep.StateBag) mu
 		err := fmt.Errorf("Error waiting for volume: %s", err)
 		state.Put("error", err)
 		ui.Error(err.Error())
+
+		// The volume ID has created, try to clean up
+		s.doCleanup = true
+		s.volumeID = volume.ID
```

Comment:
I'm not sure about this here, if the volume never transitioned to `available`, is it still created?
I understand the status could be "error" for some reason at that point, but in this case if it happens at creation, do we still need to clean it up?